doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md

```diff
@@ -237,6 +238,21 @@ Regarding the amount of information per write, please read carefully [this](http
 
 [Top](#top)
 
+####<a name="section2.3.5"></a>About the encoding
+Cygnus applies this specific encoding tailored to DynamoDB data structures:
+
+* Alphanumeric characters are not encoded.
+* Numeric characters are not encoded.
+* Underscore character, `_`, is not encoded.
+* Hyphen character, `-`, is not encoded.
+* Dot character, `.`, is not encoded.
+* Equals character, `=`, is encoded as `xffff`.
+* All other characters, including the slash in the FIWARE service paths, are encoded as a `x` character followed by the [Unicode](http://unicode-table.com) of the character.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1133/commits/3ba31a420f9ddda138c790c11f4d8fdc426ba8f4